app/templates/macros/helpers.html

```diff
@@ -19,3 +19,12 @@
   } %}
 <a href="tel:03001234931" {{attr|xmlattr}}>0300 1234 931</a>
 {%- endmacro -%}
+
+{%- macro email_address(aria_describedby=None, class=None) -%}
+  {% set attr = {
+    "aria-describedby": aria_describedby,
+    "class": class
+  } %}
+<a href="mailto:surveys@ons.gov.uk?subject=Change%20of%20details%20reference%2012346789012A" {{attr|xmlattr}}>surveys@ons.gov.uk</a>
```

Comment:
Tag needs updating to support subject rather than hardcoding test data for every instance: 

```html
<a href="mailto:surveys@ons.gov.uk{% if subject %}?subject={{ subject|urlencode }}{% endif %}"```